source/articles/2015/bundler-failing-on-el-capitan.md

```diff
@@ -0,0 +1,124 @@
+---
+title: Bundler Failing on El Capitan
+description: How to fix OpenSSL header issues after upgrading to OS X El Capitan
+
+author: Adarsh Pandit
+category: rails
+date: 2015-11-09
+logo: terminal_icon.png
+published: true
+tags: [bundler, ruby on rails, el capitan, openssl, os x, openssl]
+---
+
+After updating to El Capitan,
+installing new gems
+started to fail for what I will call
+"C-like reasons".
+Those reasons look something like this:
+
+```
+Building native extensions.  This could take a while...
+ERROR:  Error installing awesome_gem_name:
+    ERROR: Failed to build gem native extension.
+
+checking for rb_trap_immediate in ruby.h,rubysig.h... no
+checking for rb_thread_blocking_region()... no
+...
+CFLAGS=-O3 -I/Users/haven/.sm/pkg/active/include -fPIC -mmacosx-version-min=10.7 -pipe  -Wall -Wextra -Wno-deprecated-declarations -Wno-ignored-qualifiers -Wno-unused-result
+CPPFLAGS=-D_XOPEN_SOURCE -D_DARWIN_C_SOURCE -D_DARWIN_UNLIMITED_SELECT -D_REENTRANT $(DEFS) $(cppflags) -Wall -Wextra -Wno-deprecated-declarations -Wno-ignored-qualifiers -Wno-unused-result
+checking for clock_gettime()... no
+checking for gethrtime()... no
+creating Makefile
+
+make "DESTDIR=" clean
+
+make "DESTDIR="
+compiling binder.cpp
+```
+
+But here's the important part:
+
+```
+In file included from binder.cpp:20:
+./project.h:116:10: fatal error: 'openssl/ssl.h' file not found
+```
+
+## [Wha happened?](https://www.youtube.com/watch?v=k5Cfhooa1bI)
+
+With OS X El Capitan,
+Apple no longer provides OpenSSL headers.
+
+But wait, I have OpenSSL!
```

Comment:
Yes, of course. 0195bf3

[What am I, a farmer?](https://www.youtube.com/watch?v=K_P7yWnAAd0)